Source/AFError.swift

```diff
@@ -412,8 +490,12 @@ extension AFError.ResponseSerializationFailureReason {
     }
 
     var underlyingError: Error? {
-        guard case .jsonSerializationFailed(let error) = self else { return nil }
-        return error
+        switch self {
```

Comment:
Again, we should make this `switch` exhaustive to protect against future changes.

Reply:
Done.